packages/core/src/utils.ts

```diff
@@ -678,3 +679,13 @@ export function evaluateGuard<TContext, TEvent extends EventObject>(
 
   return condFn(context, _event.data, guardMeta);
 }
+
+export function toInvokeSource(
```

Comment:
a similar one is already available in `packages/core/src/invokeUtils.ts` - any reason for having both?

Reply:
That file doesn't exist, link?